pkg/k8s/apis/cilium.io/v2alpha1/bgpp_types.go

```diff
@@ -29,6 +29,10 @@ const (
 	DefaultBGPGRRestartTimeSeconds = 120
 	// BGPLoadBalancerClass defines the BGP Control Plane load balancer class for Services.
 	BGPLoadBalancerClass = "io.cilium/bgp-control-plane"
+	// PodCIDRSelectorName defines the name for a selector matching Pod CIDRs.
```

Comment:
This is a selector for IPAM pod CIDRs, correct? Might not be a bad idea to provide a bit more clarity to not confuse it with multi-pool IPAM pod CIDRs.

Reply:
Right, this is only for standard IPAM, not multi-pool. We may add multi-pool support by selecting `CiliumPodIPPool` custom resources later, after https://github.com/cilium/cilium/pull/27100 is in.